src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/StructuredTaskScope.java

```diff
@@ -342,6 +346,7 @@ private boolean isShutdown() {
      */
     private void track(Future<?> future) {
         // create the set of Futures if not already created
+        // cannot use double-checked locking since it requires use of a synchronized block
```

Comment:
I'd prefer to leave this comment as is, only because mentioning another possible implementation might confuse reader.

Reply:
Ok, removed. When/if monitors are supported properly in loom we can use double checked locking, which is more idiomatic. Hard to track these things.